WISE-Community/WISE-API
# refactor(Controllers): Pass in domain instead of Long to methods

## Changes
- Added ProjectRepository and WorkgroupRepository
- Updated tests

## Test
- Student VLE
   - save achievement (e.g. completing c-rater milestone item should enter new row in db)
   - save notification: post reply to discussion post
   - get notification: open vle
   - dismiss notification
   - Notebook
      - get/create/edit notebook items
- CM
   - retrieve achievement (e.g. open teacher report view)
- AT
   - getAuthorProjectConfig (load AT)
   - notifyAuthorBeginEnd (open/close project authoring)
   - export project
   - copyProject
   - saveProject

Closes #64
